anki/src/editor/__init__.py

```diff
@@ -205,12 +200,19 @@ def on_cloze(editor) -> None:
     editor.web.eval(f'wrap("{prefix}", "{suffix}");')
 
 
+def clear_occlusion_mode(js, note, editor):
+    return f"EditorCloset.clearOcclusionMode().then(() => {{ {js} }}); "
+
+
+def refocus(editor):
+    editor.web.setFocus()
+    editor.web.eval("EditorCloset.refocusField(); ")
```

Comment:
Thanks for testing @BEST8OY!

The bug you mentioned stems from line 208. Forcing focus on the editor seemed to be the only way to prevent following issue:

- start occlusion editor in the browser
- click on different note to load it
- the image + occlusion editor of former note is still displayed in the same field (i.e. the field content is not updated)

@hgiesel I'd be very interested if there was a less hacky way to solve this issue. I have experimented for days but didn't find a suitable solution other than setting focus.

Reply:
I'm not sure if i understood it properly.
Is the bug you mentioned in 3 steps to reproduce has been fixed by lines 208 and 209?
Because i followed your steps and the bug reproduced on my side.

1. I opened the browser.
2. Browser shows the first card. (obviously)
3. I start occlusion editor.
4. Select another card.
5. First card fields got copied into the second card.

If I don't want this problem to arise I have to add a step between steps number 1 and 2 which is selecting another card/ note other than the card that is showing by default by oppening browser.

Reply:
I think there's a misunderstanding here, the issue still exists in this PR, and @kleinerpirat asks for my help

Reply:
@kleinerpirat 
I'm not sure what is happening either.
It's a gif of what is happening.
I figured the problem is in the first card you start occlusion editor. (not first card in the browser, rather the first card you start occlusion editor for)
After first duplication I undo the changes and it doesn't happen again.
![anki_9tdT8taCQ6](https://user-images.githubusercontent.com/70865987/181045348-a0d17fb0-8e23-4172-afde-f81a1c7ed8c5.gif)

Reply:
@kleinerpirat 
I have a good news.
I disabled all my addons and enabled Closet.
Duplication is no more.
Now I have to investigate which addon is interfering with Closet.
I think PR is good to go.
Some other addon is causing this problem.